src/patterns/components/navigation/narrow.hbs

```diff
@@ -34,15 +34,15 @@ hasAngularCodeInfo: true
     </li>
 
     <li class="sprk-c-Accordion__item {{item}}">
-      <a aria-controls="details1" class="sprk-c-Accordion__summary {{summary}}" href="#">
+      <a class="sprk-c-Accordion__summary {{summary}}" href="#">
         <span class="sprk-b-TypeBodyTwo sprk-c-Accordion__heading {{heading}}">
           Item 2
         </span>
       </a>
     </li>
 
     <li class="sprk-c-Accordion__item {{item}}" data-sprk-toggle="container">
-      <a aria-controls="details1" class="sprk-c-Accordion__summary {{summary}}" data-sprk-toggle="trigger" data-sprk-toggle-type="accordion" href="#">
+      <a aria-controls="details3" class="sprk-c-Accordion__summary {{summary}}" data-sprk-toggle="trigger" data-sprk-toggle-type="accordion" href="#">
```

Comment:
I see you're removing the aria-controls from one of the accordions and updating the ids for the others. 
I think we want to just remove the use of aria-controls on all the accordions.

Reply:
@RCopeland 

That's what I thought when I first read it at first, but when I dug into the linked doc, and saw that the note on the screenshot was on the middle link--the only one that didn't open an accordion--I changed my mind about the ask.

And if we do still need `aria-controls` on items 1 and 3, then they would need those unique IDs.

@RVMendoza , could you weight in on whether my PR is what you wanted?

Reply:
@vjwilson this is it! It makes sense that the aria controls match the the corresponding IDs, thanks for the extra effort.